### ISSUE TITLE
Custom scripts run before script tags

### ISSUE THREAD

**Issue by RaitaroH** [OpenUserCSS/forums.openusercss.org#13]
When that happens the chat is also way longer than usual. The  loading/progress bar at top doesn't disappear either. I also can't send any messages.

```
ReferenceError: Raven is not defined
[Learn More]
2:1895:4
<anonymous>
https://forums.openusercss.org/user/raitaroh/chats/2:1895:4
Error: Mismatched anonymous define() module: function(){return function a(b,c,d){function e(g,h){if(!c[g]){if(!b[g]){var i="function"==typeof require&&require;if(!h&&i)return i(g,!0);if(f)return f(g,!0);var j=new Error("Cannot find module '"+g+"'");throw j.code="MODULE_NOT_FOUND",j}var k=c[g]={exports:{}};b[g][0].call(k.exports,function(a){var c=b[g][1][a];return e(c?c:a)},k,k.exports,a,b,c,d)}return c[g].exports}for(var f="function"==typeof require&&require,g=0;g<d.length;g++)e(d[g]);return e}({1:[function(a,b,c){function d(a){this.name="RavenConfigError",this.message=a}d.prototype=new Error,d.prototype.constructor=d,b.exports=d},{}],2:[function(a,b,c){var d=function(a,b,c){var d=a[b],e=a;if(b in a){var f="warn"===b?"warning":b;a[b]=function(){var a=[].slice.call(arguments),g=""+a.join(" "),h={level:f,logger:"console",extra:{arguments:a}};"assert"===b?a[0]===!1&&(g="Assertion failed: "+(a.slice(1).join(" ")||"console.assert"),h.extra.arguments=a.slice(1),c&&c(g,h)):c&&c(g,h),d&&Functi…
require.js:7:289
F
require.js:7:289
v
require.js:14:138
c
require.js:26:240
requirejs
require.js:32:230
<anonymous>
ajaxify.js:19:1
l
jquery.js:3583:20
a/</u<
jquery.js:3651:11
```

**Comment by DecentM** [comment]
Raven is Sentry's Javascript SDK. It's blocked by some default uBlock filters. Try turning them off or whitelisting sentry in the advanced settings and see if the error still persists.

**Comment by RaitaroH** [comment]
I already have uBlock off on the forums.

Not a big deal of a problem. A refresh can solve it. I just mentioned it because it may fuck up other things.

**Comment by DecentM** [comment]
That means that the custom scripts set to run run before its dependencies load. I renamed the issue to reflect that. Thanks!

**Comment by DecentM** [comment]
Solved by initializing Raven in the deferred script.
